packages/main/src/themes/sap_fiori_3_dark/Avatar-parameters.css

```diff
@@ -0,0 +1,7 @@
+:root {
```

Comment:
The Convention that is currently in use is: here in sap_fiori_3_dark/Avatar-parameters.css 
 we import the base params 
@import "../base/Avatar-parameters.css";

 and then in sap_fiori_3_dark/parameters-bundle.css  leave only the theme specific import
@import "./Avatar-parameters.css";

and removing @import "../base/Avatar-parameters.css"; from sap_fiori_3_dark/parameters-bundle.css
as it will be imported above

The way you did it, it works as well, but it's not consistent will all the other components params.